api/api.go

```diff
@@ -1098,6 +1098,11 @@ func (v *VolumeSpec) GetCloneCreatorOwnership(ctx context.Context) (*Ownership,
 
 	// If there is user information, then auth is enabled
 	if userinfo, ok := auth.NewUserInfoFromContext(ctx); ok {
+		// if a public user clones a volume, we should not update
```

Comment:
Remove this and let `OwnershipSetUsernameFromContext(ctx, nil)` handle returning the appropriate ownership object, which could be a public one or a `nil`